### ISSUE TITLE
Proton fails with Kernel 5.8.0-36 Ubuntu 20.04.1

### ISSUE THREAD

**Issue by wakemeapp** [ValveSoftware/Proton#4534]
#### Your system information

* Steam client version API Steam v020, steam package version 1608507519
* Distribution: Ubuntu 20.04.1 LTS 
* Opted into Steam client beta?: No
* Have you checked for system updates?: Yes I updated Ubuntu and Steam.
* Nvidia Drivers 455.45.01
My PC:
AMD Ryzen 7 2700X
Nvidia GTX 1080 Ti
24 GB RAM 3600 Mhz
X470 Gaming 7 Wifi Gigabyte

Yesterday, I updated my Ubuntu 20.04.1 LTS and obtained the new official kernel (with the update) 5.8.0-36.

With this kernel I have got a lot of problems with some games with Steam Proton.
For example, with the Kernel 5.4.0-59 all works well, I could play Cyberpunk 2077 and Far Cry 5 without problems.

Now, with the kernel 5.8.0-36 I can't play Cyberpunk 2077 (I can arrive to the menu, but when I continue with my saved game, it crashes) and I have got low fps in some areas with Far Cry 5, obtaining for example 7-8 fps in the cinematic scene of Faith, and sometimes the fps drops a lot.

When I open CS:GO, it closes automatically after few seconds.

Please work with Ubuntu and fix that, because some games doesn't start with the new kernel.

Regards and thank you Valve/GabeNewell for Steam Proton. Thanks.

**Comment by kisak-valve** [comment]
Hello @wakemeapp, Proton issues are being tracked in the Proton issue tracker, so I've transferred this issue report.

Cyberpunk 2077 is known unstable with nVidia's driver. Whether it works slightly better with one kernel or another is largely irrelevant to Proton until the major issues are triaged. That said, this reads like a video driver issue in general and that should be reported to your video driver vendor.

**Comment by wakemeapp** [comment]
OK, but the problems started with the new Kernel, and when I open CS:GO it crashes after few seconds.

I will try to change the Nvidia driver and watch if it works or not. Thanks.

**Comment by kisak-valve** [comment]
Sorry, but this issue really should be reported directly to nVidia. You've isolated the issue to a kernel <-> video driver interaction and that can't be fixed by a Valve or Proton dev.

Closing as not-our-bug.

**Comment by wakemeapp** [comment]
> Sorry, but this issue really should be reported directly to nVidia. You've isolated the issue to a kernel <-> video driver interaction and that can't be fixed by a Valve or Proton dev.
> 
> Closing as not-our-bug.

Thank you very much, you were right, it was a video driver issue generated due to I have got a RAM socket broken and it reads the data with errors. When I installed the new update of Ubuntu receiving the new Nvidia drivers and new kernel, my system read bad the data and generated the graphic problem because the driver was corrupted.

Thank you for the info, you helped me a lot.
